main.js

```diff
@@ -63,7 +61,7 @@ const addBook = (element) => {
 
```

Comment:
On line 55 you have your `add book` method outside of the class.

![image](https://user-images.githubusercontent.com/38088818/191612056-2b8ad663-829d-4695-8e03-4643be187ed8.png)

- Kindly make sure that your add book method is inside the class as demonstrated in the comment above.